cmd/common/params_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/params"
+)
+
+func TestHandleFileArguments(t *testing.T) {
+	t.Parallel()
+
+	pd := &params.ParamDesc{
+		Key:      "test",
+		TypeHint: "string",
```

Comment:
Shouldn't this be `TypeBytes`?

Reply:
Ah, this isn't also checking the base64 compression/decompression, sorry. Maybe there should be some test cases for that as well, though.